model/parameter_fitting.py

```
@@ -2,18 +2,19 @@
 パラメータフィッティング機能
 """
 import sys
-from typing import Tuple, Dict
+from typing import Tuple, Dict, Optional
 import pandas as pd
```

Ruff (0.12.2)

`pandas` imported but unused

Remove unused import: `pandas`

(F401)